jsonnet/cmd.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2016 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/google/go-jsonnet"
+)
+
+func usage() {
+	fmt.Println("usage: jsonnet <filename>")
+}
+
+func main() {
+	// TODO(sbarzowski) Be consistent about error codes with C++ maybe
```

Comment:
Once the code stabilizes a bit this will be a very good option (at least for issues observable by the user).